rest/src/main/resources/scripts/SDK生成器开发指南.md

```
@@ -0,0 +1,936 @@
+| `double`, `Double`            | `float64`                                 | `*float64`                                |
+| `boolean`, `Boolean`          | `bool`                                    | `*bool`                                   |
+| `Date`, `Timestamp`           | `time.Time` (param) / `ZStackTime` (view) | `*time.Time` (param) / `*ZStackTime` (view) |
+| `Enum`                        | `string`                                  | `*string`                                 |
+| `byte[]`                      | `[]int8` (Java byte 是有符号 -128~127)        | `[]int8`                                   |
+| `List<T>`                     | `[]T`                                     | `[]T`                                      |
+| `Map<String, V>`              | `map[string]V`                            | `map[string]V`                             |
+| 其他复杂类型                        | 生成对应的 struct                              | -                                           |
+| 无法识别的类型                       | `interface{}`                             | `interface{}`                              |
+
+### 3.2 上下文感知的类型映射
+
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)